wgengine/netstack/netstack.go

```diff
@@ -53,8 +54,9 @@ type Impl struct {
 	mc      *magicsock.Conn
 	logf    logger.Logf
 
-	mu  sync.Mutex
-	dns DNSMap
+	mu              sync.Mutex
+	dns             DNSMap
+	connsToSubnetIP map[tcpip.Address]int
```

Comment:
When I see something named `fooToBar` I think `map[foo]bar`, but this is "connections to" not "a mapping from X to Y", so a different "to".

Maybe `connsOpenBySubnetIP`?

But at least a comment.

Also, can we use a netaddr.IP here instead of a tcpip.Address?